ts/src/bitget.ts

```diff
@@ -3486,7 +3486,7 @@ export default class bitget extends Exchange {
         let paginate = false;
         [ paginate, params ] = this.handleOptionAndParams (params, 'fetchOHLCV', 'paginate');
         if (paginate) {
-            return await this.fetchPaginatedCallDeterministic ('fetchOHLCV', symbol, since, limit, timeframe, params, maxLimitForRecentEndpoint);
+            return await this.fetchPaginatedCallDeterministic ('fetchOHLCV', symbol, since, limit, timeframe, params, maxLimitForHistoryEndpoint);
```

Comment:
what if the user uses the recent endpoint?

Reply:
it might not make sense to use recent endpoint with pagination, because backperiod is very small for "recent" endpoints.

Reply:
@ttodua let's say users want to fetch 1m candles, wouldn't the recent endpoint be do the job? 1m candles can be a lot even in a short timespan

Reply:
ok, added "param" for it

Reply:
not sure this is the best approach because for pagination we require the param, otherwise we 'infer' it 🤔